### ISSUE TITLE
[Feature Request] Multiple account support for the same instance.

### ISSUE THREAD

**Issue by clawedcourtier** [enafore/enafore#30]
Hi, folks! Enafore has been pretty great so far. There's one feature I've been missing from Tusky, though, and that's being able to switch between multiple accounts on the same instance. Right now, Enafore allows you to switch between different accounts on _different_ instances, but if you attempt to log into the same instance on a different account, you'll get an error telling you that you're already logged in.

Incomplete list of use cases for being able to switch between multiple accounts on the same instance:

- Someone wants separate SFW and NSFW accounts.
- Someone wants one account that has open followers, and another account that requires follower approval.
- Headmates in a plural system want their own accounts.

Thank you for your consideration!

**Comment by easrng** [comment]
Yeah I definitely want to do this at some point but with how the codebase is structured right now that is not easy :(

**Comment by alexpdp7** [comment]
BTW, I've been using [the Firefox Multi-Account Containers extension](https://github.com/mozilla/multi-account-containers#readme) for a while to handle this for a while and it's working reasonably well. It's a bit ugly because I only use this for this purpose, so I have only two containers (default, and id2), and sometimes it's a bit confusing because opening a link on the second account will not be logged in, etc. in my usual sites.

But if anyone is looking for a workaround for this limitation, I can recommend this.
